src/Core/maps/src/Handlers/Map/MapHandler.Android.cs

```diff
@@ -77,56 +80,31 @@ protected override MapView CreatePlatformView()
 		public static void MapMapType(IMapHandler handler, IMap map)
 		{
 			GoogleMap? googleMap = handler?.Map;
-			if (googleMap == null)
-				return;
-
-			googleMap.MapType = map.MapType switch
-			{
-				MapType.Street => GoogleMap.MapTypeNormal,
-				MapType.Satellite => GoogleMap.MapTypeSatellite,
-				MapType.Hybrid => GoogleMap.MapTypeHybrid,
-				_ => throw new ArgumentOutOfRangeException(),
-			};
+			googleMap?.UpdateMapType(map);
```

Comment:
Moved to extension methods. On the one hand, we achieve homogeneity with other handlers code but in the other hand it is also necessary to invoke some properties update when the map is ready.